.github/workflows/frontend_checks.yml

```diff
@@ -105,12 +105,12 @@ jobs:
       - name: Format
         run: npm run format:ci
 
-      - name: Jest test
-        run: npm run test:ci
-
       - name: Build
         run: npm run build
 
+      - name: Jest test
+        run: npm run test:ci
+
```

Comment:
The fides-consent package needs to be built. And if build fails the tests are going to fail anyway.